packages/backend/src/controllers/users/users.controller.ts

```diff
@@ -34,20 +26,16 @@ export class UsersController {
     summary: 'Create a new user resource',
     description: 'User must be linked to a firebase ID',
   })
-  @ApiBadRequestResponse({ description: 'no user name provided' })
   @ApiCreatedResponse({
     description: 'User successfully created',
     type: UserResponseDto,
   })
   async create(@Body() createUserDto: CreateUserDto): Promise<UserResponseDto> {
```

Comment:
I think the firebase id should be obtained from the @User decorator? It seems unnecessary for it to be obtained from the dto as it will be in the bearer token.

Reply:
The problem with this is that you shouldn't have a firebase token passed if you haven't created a user yet. The token will get passed if an account is created with google account due to the previous guard logic where the a db entry will be created. We decided that it is better for the front-end to be handling this rather than making the guard logic really weird